process/templates/PROJECT_NAME_graduation_stage.md

```diff
@@ -1,7 +1,10 @@
 ## Project graduation application
 
+### Project has met all Incubating requirements
+  * "link to Incubating PR"
```

Comment:
Same comment as before: I think there is no need to ask people to dig this up.

Reply:
added verbiage arnaud requested